<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_ComSet</class>
 <widget class="QWidget" name="Dlg_ComSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>161</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>波特率</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>校验位</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>数据位</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>停止位</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>COM口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_3"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_4"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_5"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox_6"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(98, 152, 200);
border:rgb(255, 255, 255);
color:rgb(255, 255, 255);
border-radius:5px</string>
      </property>
      <property name="text">
       <string>获取串口</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
